about.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>SmartBuild About</title>
+    <link rel="stylesheet" href="css/about.css">
+    <script src="javascript/script.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@900&display=swap" rel="stylesheet"> 
+</head>
+<body>
+
+    <div class="social-icons-section">
+        <ul class="social-icons">
+            <li class="network-icons-list"><a href="https://twitter.com/mkaya133" target="_blank"><img src="img/twitter.svg" alt=""></a></li>
+            <li class="network-icons-list"><a href="https://www.linkedin.com/in/mert-kaya-0732b717b/" target="_blank"><img src="img/linkedin.svg" alt=""></a></li>
+            <li class="network-icons-list"><a href="https://github.com/mkaya13" target="_blank"><img src="img/github.svg" alt=""></a></li>
+            <li class="social-icon">Logout</li>
+            <li class="social-icon">My Page</li>
+            <li class="social-icon">English</li>
+        </ul>
+    </div>
+
+    <section class="main-page">
+
+        <nav>
+            <a href="#" class="toggle-button">
+                <span class="bar"></span>
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </a>
+    
+            <a href="/index.html"><img class="main-page-logo" src="img/smartbuild.jpg" alt=""></a>
```

Comment:
- [ ] As the previous reviewer mentioned all your alt tags are empty, kindly consider adding a description to the associated image with every tag so it's descriptive for screen readers and can act as a placeholder if the image is not available